libs/server/Resp/RespCommandsInfo.cs

```diff
@@ -12,21 +12,9 @@ namespace Garnet.server
     class RespCommandsInfo
```

Comment:
Looks like there was an issue with the merge here...

Reply:
Yeah, it looks a little weird that the single test failed🤔

Reply:
Well, the problem seems to be solved now